lib/gel/git_gems_manager.rb

```diff
@@ -0,0 +1,21 @@
+class Gel::GitGemsManager
+  # These gems all live in rails/rails repository.
+  META_GEMS = {
+    "activesupport" => "rails",
+    "actionpack" => "rails",
+    "actionview" => "rails",
+    "activemodel" => "rails",
+    "activerecord" => "rails",
+    "actionmailer" => "rails",
+    "activejob" => "rails",
+    "actioncable" => "rails",
+    "activestorage" => "rails",
+    "actionmailbox" => "rails",
+    "actiontext" => "rails",
+    "railties" => "rails",
```

Comment:
Hm, this doesn't seem right to me. Now Gel has to change if Rails adds another framework.